tests/test_schema_model.py

```diff
@@ -2,19 +2,17 @@
 
 import jsonschema
 
-from grout.models import RecordSchema, SchemaModel
+from grout.models import RecordSchema
 
 
 class SchemaModelTestCase(TestCase):
```

Comment:
This test case needs to be adjusted to reflect the fact that the `SchemaModel` methods now reside directly on the `RecordSchema` model.